backend/routers/index_router.py

```diff
@@ -16,3 +25,23 @@ async def healthcheck():
 async def read_main():
     logger.info("In root path")
     return {"msg": "Hello World"}
+
+
+@index_router.get("/get_my_pixel_watches", response_model=List[schemas.PixelWatch])
+def get_my_pixel_watches(user: UserResponse = Depends(get_current_user), db: Session = Depends(get_db)):
+    user_pixel_watches = db.query(PixelWatch).join(user).filter(PixelWatch.user_id == user.user.id).all()
+
+    return user_pixel_watches
+
+
+@index_router.post("/watch_my_pixel")
+async def watch_my_pixel(params: WatchPixelParams,
+                         user: UserResponse = Depends(get_current_user),
+                         db: Session = Depends(get_db)):
+    db.add(PixelWatch(latitude=params.coordinates.to_list()[0], longitude=params.coordinates.to_list()[1],
+                      datetime=params.date, user_id=user.user.id,
+                      path=params.path, row=params.row))
+    db.commit()
+    return { "latitude": params.coordinates.to_list()[0], "longitude": params.coordinates.to_list()[1],
```

Comment:
Create pydantic type for that! Also why are you returning anything? Frontend already knows latitude where user pressed